cadquery/assembly.py

```diff
@@ -400,16 +400,17 @@ def solve(self) -> "Assembly":
         constraints = []
         for c in self.constraints:
             constraints.append(((ents[c.objects[0]], ents[c.objects[1]]), c.toPOD()))
+        
+        if not constraints == []:
```

Comment:
Can't you just do `if not constraints:`?

Reply:
yeah, you can[1]. I just find this way to be more clear on what's being checked.
[1]: https://docs.python.org/3/library/stdtypes.html#truth-value-testing